backend_addon/cookiecutter.json

```diff
@@ -11,10 +11,16 @@
   "feature_headless": ["1", "0"],
   "__feature_headless": "{{ cookiecutter.feature_headless }}",
   "__feature_distribution": "0",
-  "__package_namespaces": "{{ cookiecutter.python_package_name | package_namespaces }}",
   "__package_path": "{{ cookiecutter.python_package_name | package_path }}",
   "__folder_name": "{{ cookiecutter.python_package_name }}",
   "__python_package_name_upper": "{{ cookiecutter.python_package_name | pascal_case }}",
+  "__gha_version_checkout": "v4",
+  "__gha_version_setup_uv": "v5",
+  "__gha_version_cache": "v4",
+  "__gha_python_versions": ["3.13", "3.12", "3.11", "3.10"],
+  "__gha_plone_versions": ["6.1-latest", "6.0-latest"],
+  "__gha_python_version": "3.12",
+  "__gha_plone_version": "6.1-latest",
```

Comment:
We're already prompting the user for Plone version (line 8). If they choose Plone 6.0.x then it's odd to use 6.1 for GHA.